### PR TITLE
Automate Release Workflow Setup

### PR DESCRIPTION
**Pull request by marcsanmi**
Setting up automated release workflow.

### CODE REVIEW

**Comment by marcsanmi** [comment]
Closing since pipelines aren't based on reusable workflows.

**Comment by marcsanmi** [comment]
Closing since pipelines aren't based on reusable workflows.

**Comment by paologallinaharbur** [comment]
we do have it https://github.com/newrelic/nri-mongodb/blob/master/.github/workflows/trigger_prerelease.yml

File: .github/workflows/automated_release.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: Automated release creation
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: ""
```
[the remark this answers is not in the document]
Comment (paologallinaharbur) [reply .github/workflows/automated_release.yaml:6]: This is an old PR I've reopened, Mark crated already a second one that was merged

**Pull request closed without merging**